cheeseshop/games/csgo.py

```diff
@@ -150,6 +150,12 @@ def add_routes(self, router):
                        self._handle_gsi_deathlog)
         router.add_get('/games/csgo/gsi/sources/{streamer_uuid}/moneylog',
                        self._handle_gsi_moneylog)
+        router.add_get('/games/csgo/historical/entry_kills_heatmaps',
+                       self._handle_historical_entry_kills_heatmap)
+        router.add_get('/games/csgo/historical/entry_kills_heatmaps',
+                       self._handle_historical_entry_kills_heatmap)
+        router.add_get('/api/games/csgo/historical/entry_kills',
```

Comment:
can we replace 'historical' with 'replay' or 'events'? I also think entry_kills_heatmaps is probably too specific, maybe just 'location_events'? I think we want to use GET args to actually filter what kind of events we get. e.g. location_events?ev_types=deaths&custom_filter=first_in_game for getting entry kills